network_wrangler/Scenario.py

```diff
@@ -38,14 +38,29 @@ def create_scenario(base_scenario: dict, card_directory: str = '', tags: [str] =
         base_scenario: the base scenario
         tags: only project cards with these tags will be read/validated
         folder: the folder location where the project cards will be
+        project_cards_names: project names
         '''
-        scenario = Scenario(base_scenario, project_cards = project_card_list)
 
-        if card_directory:
-            scenario.add_project_cards(card_directory, tags = tags)
+        project_cards = []
 
-        return scenario
+        prereq_dict   = OrderedDict()
+        coreq_dict    = OrderedDict()
+        conflict_dict = OrderedDict()
+
+        for project_name in project_card_names:
+            card = ProjectCard.read(os.path.join(card_directory, project_name + '.yml'))
```

Comment:
this thru line 56 should probably be part of a generic function "add project card" so that it can be called after the scenario is already created.

Reply:
I do agree, there needs to be a function to be able to "add project card(s)" after the scenario is already created. Will work on how that function will be designed after I get the rest of the skeleton for this correct.

Reply:
The add project cards method can be used for both the initial add as well as any subsequent one.